bundles/org.openhab.binding.deconz/src/main/resources/OH-INF/thing/sensor-thing-types.xml

```diff
@@ -491,6 +491,41 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+
+	<thing-type id="airqualitysensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="deconz"/>
+		</supported-bridge-type-refs>
+		<label>Air quality Sensor</label>
+		<description>A air quality sensor</description>
+		<channels>
+			<channel typeId="airquality" id="airquality"/>
+			<channel typeId="airqualityppb" id="airqualityppb"/>
+			<channel typeId="last_updated" id="last_updated"/>
+		</channels>
+
+		<representation-property>uid</representation-property>
+
+		<config-description-ref uri="thing-type:deconz:sensor"/>
+	</thing-type>
+
+	<channel-type id="airquality">
+		<item-type>String</item-type>
+		<label>Air quality level</label>
+		<description>Current air quality level</description>
+		<category>Airquality</category>
+		<state readOnly="true" pattern="%s"></state>
+	</channel-type>
+
+	<channel-type id="airqualityppb">
+		<item-type>Number</item-type>
+		<label>Air quality in ppb</label>
+		<description>Current air quality ppb (parts per billion)</description>
+		<category>Airquality</category>
```

Comment:
Which kind of particles are measured? If CO2 consider category **carbondioxide** - I don't think category Airquality exists?

Reply:
I have removed the category, at the moment there is no matching one for it. Te air quality based on measurements of volatile organic compounds (VOCs). The measured value is specified in ppb (parts per billion).